server/Html.tsx

```diff
@@ -101,7 +101,7 @@ const Html = (props: Props) => {
 				className={classNames(
 					props.bodyClassPrefix && `${props.bodyClassPrefix}-body-wrapper`,
 					getActiveSlugClassName(props.initialData, props.viewData),
-					getUserClassName(props.initialData),
+					{ ...getUserClassName(props.initialData) },
```

Comment:
not sure why this wasn't flagged earlier but had to spread classnames bc return type from `getUserClassName` was of type String[]